docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 app:
-  image: golang:1.10.4-stretch
+  image: golang:1.12.1-stretch
```

Comment:
Nitpick: this will run on `1.12.1`, while Buildkite (as well as Dockerfile below) will give you `1.12.5` once it expands `1.12.x`.